### PR TITLE
MINOR: Update verbiage on landing page https://kafka.apache.org/index.html

### PR DESCRIPTION
**Pull request by joel-hamill**

### CODE REVIEW

**Comment by joel-hamill** [comment]
ping for review @guozhangwang @dguy

**Comment by dguy** [comment]
as per: https://github.com/apache/kafka-site/pull/78#issuecomment-329644776

**Comment by joel-hamill** [comment]
@dguy i don't think these files are in https://github.com/apache/kafka/tree/trunk/docs. these are changes specifically to the header/footer/nav, and files that are specific to the kafka-site repo.

File: index.html [hunk 1]
```diff
@@ -7,17 +7,17 @@
 			<a href="/documentation/#producerapi" class="desc-item">
 				<h2>Publish &amp; subscribe</h2>
 				<p>to streams of data like a messaging system</p>
-				<span class="desc-item__cta">Learn more &raquo;</span>
+				<span class="desc-item__cta">Learn about APIs &raquo;</span>
```
Comment (guozhangwang) [inline index.html]: `Learn about producer and consumer`?

File: index.html [hunk 2]
```diff
@@ -7,17 +7,17 @@
...
 			</a>
 			<a href="/documentation/streams" class="desc-item">
 				<h2>Process</h2>
 				<p>streams of data efficiently and in real time</p>
-				<span class="desc-item__cta">Learn more &raquo;</span>
+				<span class="desc-item__cta">Learn about Streams &raquo;</span>
```
Comment (guozhangwang) [inline index.html]: `Learn about Streams API`?

File: index.html [hunk 3]
```diff
@@ -7,17 +7,17 @@
...
 			</a>
 			<a href="/intro#kafka_storage" class="desc-item">
 				<h2>Store</h2>
 				<p>streams of data safely in a distributed replicated cluster</p>
-				<span class="desc-item__cta">Learn more &raquo;</span>
+				<span class="desc-item__cta">Learn about Storage &raquo;</span>
```
Comment (guozhangwang) [inline index.html]: `Learn about storage architecture design`?

File: includes/_nav.htm [hunk 4]
```diff
@@ -11,6 +11,12 @@
         <a class="nav__item nav__sub__item" href="/documentation#gettingStarted">getting started</a>
         <a class="nav__item nav__sub__item" href="/documentation#api">APIs</a>
         <a class="b-nav__streams nav__item nav__sub__item" href="/documentation/streams">kafka streams</a>
+        <!-- Streams child pages -->
```
Comment (guozhangwang) [inline includes/_nav.htm]: Could you paste a screenshot of this change?

Comment (joel-hamill) [reply includes/_nav.htm]: @guozhangwang here you go: 
image

**Comment by dguy** [comment]
@miguno

**Comment by joel-hamill** [comment]
@derrickdoo ping ^

**Comment by joel-hamill** [comment]
after talking with @derrickdoo, I have removed the changes to the left-hand nav. @guozhangwang

**Comment by miguno** [comment]
Could you please edit/update the PR title then?

I understand that this PR is now a change to the content pane of the main page (kafka.apache.org) only?

**Comment by joel-hamill** [comment]
@miguno done https://github.com/apache/kafka-site/pull/77#issuecomment-333752239

**Review by miguno — APPROVED** [review]
LGTM.

Side note: We still need to improve the main landing page beyond what we cover in this PR.  For example, we don't mention Connect anywhere / the functionality to easily connect other systems with Kafka via Connect and connectors.

**Comment by joel-hamill** [comment]
ping @guozhangwang for merge

**Comment by guozhangwang** [comment]
LGTM. Merged to asf-site.

**Pull request closed without merging**